conf/httpd.conf-example

```diff
@@ -34,3 +34,17 @@ WSGIDaemonProcess example.com \
 WSGIProcessGroup example.com
 
 WSGIScriptAlias / /data/vhost/example.com/sayit.mysociety.org/sayit_mysociety_org/wsgi.py
+
+<Directory /data/vhost/example.com/uploads>
```

Comment:
Rather than remove the example password bit, I've updated it and re-organised the file along the lines you suggest - you are right I think, it is clearer to keep related items together.